multisig/proposals/vip-002/vip-002-ethereum.ts

```diff
@@ -0,0 +1,503 @@
+      params: [ZERO_ADDRESS, "setLastRewardingBlock(address[],uint32[],uint32[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updateJumpRateModel(uint256,uint256,uint256,uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: vWBTC_Core,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE900_JUMP7500_KINK7500],
+    },
+    {
+      target: vWETH_Core,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE900_JUMP7500_KINK8000],
+    },
+    {
+      target: vUSDC_Core,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE750_JUMP8000_KINK8000],
+    },
+    {
+      target: vUSDT_Core,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE750_JUMP8000_KINK8000],
+    },
+    {
+      target: vcrvUSD_Core,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE750_JUMP8000_KINK8000],
+    },
+    {
+      target: vcrvUSD_Curve,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE0_SLOPE750_JUMP8000_KINK8000],
+    },
+    {
+      target: vCRV_Curve,
+      signature: "setInterestRateModel(address)",
+      params: [IR_BASE200_SLOPE2000_JUMP30000_KINK5000],
+    },
+    { target: POOL_REGISTRY, signature: "acceptOwnership()", params: [] },
+    { target: COMPTROLLER_CORE, signature: "acceptOwnership()", params: [] },
+    { target: DEFAULT_PROXY_ADMIN, signature: "upgrade(address,address)", params: [POOL_REGISTRY, POOL_REGISTRY_IMPL] },
+    {
+      target: COMPTROLLER_CORE,
+      signature: "setPriceOracle(address)",
+      params: [RESILIENT_ORACLE],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: ["Core", COMPTROLLER_CORE, "500000000000000000", "1100000000000000000", "100000000000000000000"],
+    },
+    { target: COMPTROLLER_CURVE, signature: "acceptOwnership()", params: [] },
+    {
+      target: COMPTROLLER_CURVE,
+      signature: "setPriceOracle(address)",
+      params: [RESILIENT_ORACLE],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: ["Curve", COMPTROLLER_CURVE, "500000000000000000", "1100000000000000000", "100000000000000000000"],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WBTC, "29818818", ETHEREUM_MULTISIG],
+    },
+    {
+      target: WBTC,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, 0],
+    },
+    {
+      target: WBTC,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, "29818818"],
+    },
+    {
+      target: vWBTC_Core,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["6171"],
```

Comment:
Assuming 12 seconds per block on Ethereum, in 24 hours there would be 7,200 blocks. The previous calculation was assuming 14 seconds per block.

```suggestion
      params: ["7200"],
```